quickstart/conf/hive.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: hive.stackable.tech/v1alpha1
+kind: HiveCluster
+metadata:
+  name: simple-derby
+spec:
+  version: 2.3.9
+  metastore:
+    roleGroups:
+      default:
+        selector:
+          matchLabels:
+            kubernetes.io/os: stackable-linux
```

Comment:
For pure kubernetes this would be: `kubernetes.io/os: linux`
(Not sure if this documents the "older" stuff working with agent - then its ok)